jdaviz/configs/mosviz/tests/test_data_loading.py

```diff
@@ -221,12 +221,12 @@ def test_load_single_image_multi_spec(mosviz_helper, mos_image, spectrum1d, mos_
     assert label_mouseover.icon == ''
 
     label_mouseover._viewer_mouse_event(spec2d_viewer,
-                                        {'event': 'mousemove', 'domain': {'x': 10, 'y': 100}})
+                                        {'event': 'mousemove', 'domain': {'x': 100, 'y': 10}})
 
     # Note: spectra2d Wave loaded in meters, but we respect one spectral unit, so the meters in
     # converted to Angstrom (the spectra1d spectral unit).
-    assert label_mouseover.as_text() == ('Pixel x=00010.0 y=00100.0 Value +8.12986e-01 Jy',
-                                         'Wave 1.10000e+05 Angstrom', '')
+    assert label_mouseover.as_text() == ('Pixel x=00100.0 y=00010.0 Value +1.26544e-01 Jy',
+                                         'Wave 1.01000e+06 Angstrom', '')
```

Comment:
is this transpose expected or fixing a previous bug?

Reply:
Fixing a previous bug, the data shape was backwards compared to what it should have been in the test fixture given the WCS definition.

Reply:
That's the reason for the changes in  `jdaviz/configs/mosviz/tests/test_tools.py` as well.